scripts/aws/cfn/microservice-dev-cfg.json

```diff
@@ -38,6 +38,7 @@
     "MaxTasksNumber": "4",
     "AutoscalingThreshold": "1500",
     "CpuValue": "2048",
-    "MemoryAmount": "4GB"
+    "MemoryAmount": "4GB",
+    "MountEfs": "true"
```

Comment:
@fdammassa @mbrachi dubbio: va bene così oppure in DEV lasciamo a false? (mentre in hotfix e prod sarà true?)

Reply:
@vincenzo-racca-pa questa riga toglila, la mettiamo a true solo in prod

Reply:
@vincenzo-racca-pa quel parametro e' alla bisogna, nel senso che va messo a true ogni qual volta serve montare l'EFS, in hotfix e prod sara' sempre a "false" e verra' valorizzato sempre se si ha necessita' di un dump.
Sicuramente in prod servira' e ci pensiamo noi da pn-configuration.